section3/reflection.md

```diff
@@ -2,12 +2,30 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
-1. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+- It was hard to understand what you asked. By watching videos and asking to friends, I tried to do something. While searching , I have found different ways how to write my codes.But I am still spending more time until figure out the problems by myself.
 
-1. In your own words, explain what an Array method is.
+2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
-1. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+- **Array list** is used in Netflix by showing movie types by developers.
 
-1. If you had to teach a 5 year old about loops, how would you explain it?
+3. In your own words, explain what an Array method is.
+- ###### Array Methods are used for adding or removing something (one or more than one items of String,numbers or boolean data types) from Array.
 
-1. What questions do you still have about Arrays and loops?
+- **Unshift**
+This method helps us to add one or more than one item , in the beginning of array.
+varName.**unshift();**
+
+4. What is the purpose of a loop? When we would use a loop in conjunction with array vs.
+without? loopu ne zaman array
+
+- **Loops** is used for iteration based on condition until the condition is evaluated false.
+Array and loop are used together to sort many different types of data, iterate through indexes in that array, and add new information while repeating.
+
+5. If you had to teach a 5 year old about loops, how would you explain it?
+
+- While a kid is studying on numbers , I can show  how to write numbers between 1 and 10 ,
+instead of writing all number between 1 to 10 .
+
+6. What questions do you still have about Arrays and loops?
+
+- I searched and tried but I couldn't find answer for showing "i*=2" or divide / , instead of "i++"!!! how to show divide or times?
```

Comment:
The `++` increment operator is a special shortcut of `i += 1` in JavaScript (which is also a shortcut of `i = i + 1`.

While `i *= 2` and `i /= 2` are valid JS, I don't think there are comparable shortcuts to `i++`.